apis/r/DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: TileDB SOMA
 Description: Interface for working with 'TileDB'-based Stack of Matrices,
     Annotated ('SOMA'): an open data model for representing annotated matrices,
     like those commonly used for single cell data analysis.
-Version: 1.4.0.1
+Version: 1.4.2
```

Comment:
I wondered about just that yesterday.

Reply:
@eddelbuettel it was a mistake on my part. I forgot to bump it here for 1.4.1. And tags are immutable.

Reply:
@johnkerl But we are not talking about changing a tag.  We are just talking of filling in the missing half-step the R package did not do to reach 1.4.1.  Anyway.